bgpd/bgp_labelpool.c

```diff
@@ -607,3 +608,174 @@ void bgp_lp_event_zebra_up(void)
 		skiplist_delete_first(lp->inuse);
 	}
 }
+
+DEFUN_HIDDEN(show_bgp_labelpool_summary, show_bgp_labelpool_summary_cmd,
```

Comment:
I don't think this needs to be hidden.  We have `show thread cpu` `show mem` `show work-queues` `show thread poll` `show modules` `show debugging hashtable`

Reply:
ack - will unhide and document.

Reply:
done